topmenubar/src/main/java/de/interseroh/tmb/client/TopMenuBarWebApp.java

```diff
@@ -230,4 +248,11 @@ private RootPanel getWidgets(String element) {
 		return RootPanel.get(element);
 	}
 
+	private void handleCookies() {
+		Collection<String> cookieNames = Cookies.getCookieNames();
+
+		for (String cookieName : cookieNames) {
+			logger.info("I found "+cookieName+" value "+Cookies.getCookie(cookieName));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)